config.toml.example

```diff
@@ -0,0 +1,633 @@
+# ==============================================================================
+#     _    ____  ____ _  _____ _____
+#    / \  / ___|/ ___| |/ /_ _|_   _|
+#   / _ \| |  _| |  _| ' / | |  | |
+#  / ___ \ |_| | |_| | . \ | |  | |
+# /_/   \_\____|\____|_|\_\___| |_|
+#
+# This is a reference config file used for Aggkit
+# setup.
+# The below configs are the default mandatory parameters to be used.
+# ==============================================================================
+
+PathRWData = "<PATH>"
+L1URL = "<URL>"
+L2URL = "<URL>"
+
+AggLayerURL = "<URL>"
+
+AggchainProofURL= "<URL>"
+
+NetworkID = <NETWORK_ID>
+
+SequencerPrivateKeyPath = "<PATH>"
+SequencerPrivateKeyPassword  = "<PASSWORD>"
```

Comment:
As a side note, unrelated to this PR, but probably it would make sense to rename this on some occasion

```suggestion
AggSenderPrivateKeyPath = "<PATH>"
AggSenderPrivateKeyPassword  = "<PASSWORD>"
```